src/sockets.js

```diff
@@ -2,19 +2,28 @@ import { SOCKET_PING } from 'syft-helpers.js';
 
 export default class Sockets {
   constructor(opts) {
-    const { url, logger, instanceId, onOpen, onClose, onMessage } = opts;
+    const {
+      url,
+      logger,
+      instanceId,
+      onOpen,
+      onClose,
+      onMessage,
+      keepAliveTimeout
```

Comment:
Actually, it would be better if we did:

```js
onClose,
onMessage,
keepAliveTimeout = 20000
```

You can assign defaults to your arguments this way, rather than doing it below (on line 18, which you can now remove).